runner/src/main/scala/stress/config/RunnerConfig.scala

```diff
@@ -0,0 +1,97 @@
+package stress.config
+
+import java.util.concurrent.TimeUnit.MILLISECONDS
+
+import com.typesafe.config.ConfigFactory
+import com.typesafe.scalalogging.LazyLogging
+
+import scala.concurrent.duration.{FiniteDuration, Duration}
+
+/**
+ * When adding a new configuration option don't forget to check that it has the same path at `runner.conf`.
+ * If you add a duration, use [[RunnerConfig.duration()]] for accessing it.
+ */
+object RunnerConfig extends LazyLogging {
+
+  private val CONFIG_OBJECT_PATH = "xl.runner"
+
+  lazy private val rootConfig = ConfigFactory.load("runner.conf")
+
+  lazy private val runnerConfig = rootConfig.getConfig(CONFIG_OBJECT_PATH)
+
+  private val durationDilation = runnerConfig.getDouble("durationDilation")
+
+  /**
+   * This object contains public, user-facing config parameters that should work just with `-D`.
+   * E.g.: -Dusers=10,
+   */
+  object public {
+
+    val users = rootConfig.getInt("users")
+
+    def baseUrls: List[String] = rootConfig.getString("baseUrl").split(",").toList
```

Comment:
But would it require brackets around? We need to override it via command line like this: `-DbaseUrl=http://localhost:4516,http://google.com`